src/main/steps/common/keep-details-private/KeepDetailsPrivatePostController.ts

```diff
@@ -80,11 +81,15 @@ export class KeepDetailsPrivatePostController extends PostController<AnyObject>
 
     const caseData = toApiFormat(req?.session?.userCase);
     caseData.id = caseReference;
+    const accessCode = req.session.userCase.caseInvites?.filter(
```

Comment:
@hmcts-abduldaim Please delete this line, we don't send access code for case update.

Reply:
it's been removed